docs/source/welcome.ipynb

```diff
@@ -53,27 +41,31 @@
     "- Accessing\n",
     "\n",
     "    - How to Access Data from the MAAP\n",
-    "    \n",
-    "- Processing\n",
-    "    \n",
-    "    - How to Create a Normalized Difference Vegetation Index (NDVI) Cloud Optimized GeoTIFF (COG)\n",
+    "    - How to Access EDAV Data via Web Coverage Service\n",
     "    \n",
     "- Querying\n",
     "\n",
-    "    - How to Query Data from the MAAP via MAAP API\n",
     "    - How to Query Data from the MAAP via Python Client\n",
+    "    - Fields Within the Gedi Cal/Val Data\n",
+    "    - How to Query ATL08 Entwine Point Tiles\n",
     "    \n",
-    "- Sharing\n",
+    "- User Data\n",
     "\n",
     "    - How to Share Data Products to the MAAP Data Store\n",
     "\n",
-    "**Help and Support**\n",
+    "**FAQ**\n",
```

Comment:
I think this should be broken up into:

**Registering an algorithm**
- How Do I Register My Algorithm to MAS?
- How Do I Delete My Algorithm from MAS?
- How Do I View Algorithms Already Registered in MAS?

**Running a Data Processing System (DPS) Job**
- How Do I Run a DPS Job?
- How Do I Check the Status of My DPS Job?
- How Do I Get the Results from My DPS Job?

**Frequently Asked Questions (FAQs)**
- How Do I Name My Workspace Something Useful?
- How Do I Share My Workspace with Another MAAP User?
- How Do I Copy My EarthData Search into My Jupyter Notebook?
- How Do I Import Granules over from My EarthData Search into My Jupyter Notebook?
- Which Files in My Workspace Are Persistent, and Which Ones Are Only Available in My Workspace?